x/evm/keeper/keeper.go

```diff
@@ -28,6 +29,7 @@ type Keeper struct {
 	CommitStateDB *types.CommitStateDB
 	TxCount       *count
 	Bloom         *big.Int
+	Logs          []*ethtypes.Log
```

Comment:
```suggestion
```

Continuation of previous, I don't see how this has any use. If you are trying to cache the logs, there is already a cache used in the CommitStateDB

Reply:
yeah I see. would it be of any use to save all the logs in a block together?

Reply:
seems redundant, you'd be storing two copies of every log, one indexed by tx hash and one indexed by block hash. If you ever do need all by block you could just get all tx hashes and just query each or have a new endpoint to do that all in one pass